lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -1135,8 +1174,9 @@ def is_text(cmd):
 
 
 def is_local_nxapi(module):
-    transport = module.params['transport']
-    provider_transport = (module.params['provider'] or {}).get('transport')
+    transport = module.params.get('transport')
```

Comment:
@trishnaguha Fixed this in an open PR.  I needed this patch.

https://github.com/ansible/ansible/pull/59031/files#diff-367b0badce453463ffc33218f1507bdbR1138

Reply:
ack